src/env/dev/terraform.tfvars

```diff
@@ -322,3 +323,8 @@ gpd_plan_sku_size = "S1"
 reporting_function_autoscale_minimum = 1
 reporting_function_autoscale_maximum = 3
 reporting_function_autoscale_default = 1
+
+// GPD Payments
+payments_always_on            = false
+payments_paa_id_intermediario = "77777777777"   // TODO
```

Comment:
@corvettafabio we need this configuration information in dev/uat/prod env for GDP/PAYMENT service as PT.

cc @jacopocarlini